ansible/host_vars/localhost/python.yml

```
@@ -0,0 +1 @@
+ansible_python_interpreter: $LOCALHOST_PYTHON
```

For me, this fails with:
```
PLAY [localhost] **********************************************

TASK [Gathering Facts] **********************************************
fatal: [localhost]: FAILED! => {"ansible_facts": {}, "changed": false, "failed_modules": {"setup": {"failed": true, "module_stderr": "", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 0}}, "msg": "The following modules failed to execute: setup\n"}

PLAY RECAP **********************************************
localhost                  : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
```

What you probably want to do is
```suggestion
ansible_python_interpreter: "{{ lookup('env','LOCALHOST_PYTHON') }}"
```
or maybe even add a default in addition, if you think it makes sense:
```
ansible_python_interpreter: "{{ lookup('env','LOCALHOST_PYTHON') | default('/some/path') }}"
```